docs/_docs/04-development.md

```diff
@@ -317,7 +317,9 @@ class ChiHousingSpider(Spider):
     # ...
 ```
 
-Often a value for meetings returned by a spider will be the same regardless of meeting content (an example is that most meetings will always have `False` for the `all_day` value). In this case, feel free to remove the `_parse_*` method for that field, and simply include the value in each dictionary (so `'all_day': False` in this example rather than `'all_day': self._parse_all_day(item)`).
+Often a value for meetings returned by a spider will be the same regardless of meeting content (an example is that most meetings will always have `False` for the `all_day_value`). For fields like `classification`, `all_day`, and `title` (sometimes), feel free to remove the `_parse_*` method for that field, and simply include the value in each dictionary (so `'all_day': False` in this example rather than `'all_day': self._parse_all_day(item)`).
```

Comment:
Small thing here, but I'd change "`all_day_value`" to "`all_day` value"

Reply:
Oops! Good catch.

Reply:
Fixed in the next commit.